app/views/validation_requests/index.html.erb

```diff
@@ -56,8 +56,18 @@
                   <td class="govuk-table__cell limit-column-width">
                     <%= request_closed_at(validation_request)  %>
                   </td>
-                  <td class="govuk-table__cell limit-column-width">
-                    <%= applicant_response(validation_request) %>
+                   <td class="govuk-table__cell limit-column-width">
+                    <% if validation_request.is_a?(AdditionalDocumentValidationRequest) %>
+                      <ul id="additional-documents" class="govuk-list">
+                        <% validation_request.documents.each do |document| %>
+                          <%= content_tag(:li, id: dom_id(document)) do %>
```

Comment:
nice one using content tags! I just learnt about this little dom_id magic, but after reading on it it seems from rails 5 you can also just call this a "tag" element, so this content_tag can be replaced with 
```
       <%= tag.validation_request do %>
       ```
and same thing with ids shows up on the dom! 
[https://www.justinweiss.com/articles/the-lesser-known-features-in-rails-5-dot-1/]

Reply:
Nice. To replicate what there was I think I need to do `<%= tag.li id: dom_id(document) do %>`

Reply:
I gave it a go with the little snippet above and it auto generated ids so that may be all you need